scylla-macros/src/parser.rs

```diff
@@ -0,0 +1,21 @@
+use proc_macro::TokenStream;
+use syn::{DeriveInput, parse2, Data, Fields, FieldsNamed};
+use proc_macro2::Ident;
+
+/// Parsers the tokens_input to a DeriveInput and returns the struct name from which it derives and
```

Comment:
Nit: s/Parsers/Parses

Reply:
You mean a new module called Parsers?

Reply:
@piodul  This is the only comment I haven't processed in the new PR: https://github.com/scylladb/scylla-rust-driver/pull/139. What do you exactly mean?

Reply:
I just meant to say that there is a typo, and `Parsers` should be changed to `Parses` (without the "r").

`s/Parsers/Parses` is the `sed` syntax for changing the first occurrence of `Parsers` into `Parses`. Sorry, I shouldn't have assumed that everybody knows this tool.